### ISSUE TITLE
Salesforce mapping from multi-select field

### ISSUE THREAD

**Issue by Mosnar** [verbb/formie#789]
**Description**

It looks like there's an issue mapping a multi-value field from Formie to Salesforce. Currently seeing this error:
```
[{"message":"Cannot deserialize instance of multipicklist from START_ARRAY value [line:1, column:291]","errorCode":"JSON_PARSER_ERROR"}]
```

It looks like Salesforce is expecting multipicklist values to be a string separated by semicolons rather than an array: 
https://developer.salesforce.com/forums/?id=906F00000008rhVIAQ

**Additional info**
- Plugin version: 1.5.10
- Craft version: 3.7.36
- Multi-site: No

**Comment by engram-design** [comment]
Yup, looks like you're totally right. Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.10",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.11](https://github.com/verbb/formie/releases/tag/1.5.11)
